assets/l10n/intl_en.arb

```diff
@@ -2758,7 +2758,7 @@
   "@askToInvite": {},
   "select": "Select",
   "@select": {},
-  "copyMessageText": "Copy message text",
+  "copyMessageText": "Copy message",
```

Comment:
why need this change? we only copy text?

Reply:
Because this message is for copying message for all types of messages (text, image, video, ...). So change to copy message only is better

Reply:
back to "copy message text"